components/search/FacetDropdown.vue

```diff
@@ -0,0 +1,253 @@
+<template>
+  <b-dropdown
+    ref="dropdown"
+    v-click-outside="applySelection"
+    :variant="hasSelection"
+    class="mr-2"
+    :data-type="facetType"
+    data-qa="search facet"
+  >
+    <template v-slot:button-content>
+      <span :data-qa="`${facetName} dropdown button`">
+        {{ facetName }}
+      </span>
+    </template>
+
+    <b-dropdown-form class="options-container">
+      <div
+        v-for="(option, index) in sortOptions"
+        :key="index"
+      >
+        <b-form-radio
+          v-if="isRadio"
+          :id="`${option}_${RADIO}`"
+          v-model="radioSelected"
+          :value="option"
+          :data-qa="`${option} ${RADIO}`"
+          @input="applySelection"
+        >
+          {{ $t(`facets.${facet.name}.options.${option}`) }}
+        </b-form-radio>
+
+        <b-form-checkbox
+          v-else
+          :id="`${option.label}_checkbox`"
+          v-model="preSelected"
+          :value="option.label"
+          :data-qa="`${option.label} checkbox`"
+          :class="{ 'is-selected' : selectedFacet.some(s => s === option.label) }"
```

Comment:
I think you can use the Bootstrap class "font-weight-bold" directly and get rid of the is-selected class here and in the style below.